src/ALE/coord_hycom.F90

```diff
@@ -69,11 +69,13 @@ subroutine end_coord_hycom(CS)
 end subroutine end_coord_hycom
 
 !> This subroutine can be used to set the parameters for the coord_hycom module
-subroutine set_hycom_params(CS, max_interface_depths, max_layer_thickness, interp_CS)
+subroutine set_hycom_params(CS, max_interface_depths, max_layer_thickness, interp_CS, ref_pressure)
```

Comment:
This new argument is not used. It needs a
```if (present(ref_pressure)) CS%ref_pressure = ref_pressure```
added

Reply:
Thanks @adcroft

Reply:
Looks like it might be easier to not add this new arg?

Reply:
Given that the hycom CS doesn't have a ref_pressure to change, perhaps easier to drop this new argument. The PR is asimed at flexible diagnostics and the hybrid coordinates should not be used for diagnostics.

Reply:
That's a fair argument. Let me do a more careful compilation test this time -_-. Serves me right for trying to submit a PR before vacation.